docs/content/en/running/upgrading.md

```diff
@@ -120,6 +120,30 @@ If you're using docker:
 
 This can take a while depending on your instance size.
 
+Upgrading to DefectDojo Version 2.0.x.
+--------------------------------------
+
+WARNING: Upgrade to 1.15.x first before upgrading to 2.0.0, otherwise you will brick you instance.
+
+We decided to name this version 2.0.0 because we did some big cleanups in this release:
+
+- Remove API v1
+- Remove setup.bash installation method
+- Rename Fidning.is_Mitigated field to Finding.is_mitigated
+- Remove everything related to the old tagging library
+
```

Comment:
could we add the `numerical severity` notation ?

Reply:
done